app/src/main/java/co/krypt/kryptonite/pairing/PairDialogFragment.java

```diff
@@ -62,21 +62,14 @@ private Dialog createPairingDialog(final PairFragment pairFragment) {
                 .setPositiveButton("Pair", new DialogInterface.OnClickListener() {
                     public void onClick(final DialogInterface dialog, int id) {
                         if (getTargetFragment() instanceof PairListener) {
-                            Runnable onPair =
-                                    new Runnable() {
-                                        @Override
-                                        public void run() {
-                                            final PairListener listener = (PairListener) getTargetFragment();
-                                            new Thread(new Runnable() {
-                                                @Override
-                                                public void run() {
-                                                    //TODO: detect existing pairings
```

Comment:
I've added it back as a new commit. Tell me if you'd want it integrated into the original commit - piece of cake to do, but the PR will basically tell you that I remade *all* the commits, which is a bit annoying for the reviewer.